app/server/index.js

```diff
@@ -48,6 +50,24 @@ if (process.env.PGPORT) {
 databaseURL += '/';
 databaseURL += process.env.PGDATABASE || 'ggircs_dev';
 
+// Graphile-worker function
+async function worker() {
+  // Run a worker to execute jobs:
+  await run({
+    connectionString: databaseURL,
+    concurrency: 5,
+    pollInterval: 1000,
+    taskDirectory: path.resolve(__dirname, 'tasks')
+  });
+}
+
+// Start graphile-worker
+worker().catch(error => {
+  if (error) {
+    throw error;
+  }
+});
```

Comment:
```suggestion
if (!process.env.CI) {
  worker().catch(error => {
    if (error) {
      throw error;
    }
  });
}
```

Reply:
I suspect that the e2e tests are failing because `.env` is missing and requires the SMTP string. An alternative to disabling the worker is to add our mailtrap credentials and create that `.env` file before running the e2e tests. That could be in a future PR though.

Reply:
Relates to environment-specific cascading config defaults; see note about mozilla/node-convict